src/format/strftime.rs

```diff
@@ -518,3 +518,42 @@ fn test_strftime_docs() {
     assert_eq!(dt.format("%n").to_string(), "\n");
     assert_eq!(dt.format("%%").to_string(), "%");
 }
+
+#[cfg(all(feature = "locales", test))]
+#[test]
+fn test_strftime_docs_localized() {
+    use pure_rust_locales::Locale;
+    use {FixedOffset, TimeZone, Timelike};
+
+    let dt = FixedOffset::east(34200).ymd(2001, 7, 8).and_hms_nano(0, 34, 59, 1_026_490_708);
+
+    // date specifiers
+    assert_eq!(dt.format_localized("%b", Locale::fr_BE).to_string(), "jui");
+    assert_eq!(dt.format_localized("%B", Locale::fr_BE).to_string(), "juillet");
+    assert_eq!(dt.format_localized("%h", Locale::fr_BE).to_string(), "jui");
+    assert_eq!(dt.format_localized("%a", Locale::fr_BE).to_string(), "dim");
+    assert_eq!(dt.format_localized("%A", Locale::fr_BE).to_string(), "dimanche");
+    assert_eq!(dt.format_localized("%D", Locale::fr_BE).to_string(), "07/08/01"); // TODO
+    assert_eq!(dt.format_localized("%x", Locale::fr_BE).to_string(), "07/08/01"); // TODO
+    assert_eq!(dt.format_localized("%F", Locale::fr_BE).to_string(), "2001-07-08"); // TODO
+    assert_eq!(dt.format_localized("%v", Locale::fr_BE).to_string(), " 8-jui-2001"); // TODO
```

Comment:
While writing the test I realized there are still a lot of things wrong. Not all languages format their date and time the same way than in english. Those should use D_FMT, T_FMT and D_T_FMT https://docs.rs/pure-rust-locales/0.5.2/pure_rust_locales/POSIX/LC_TIME/index.html